src/lwc/geFormFieldPicklist/geFormFieldPicklist.js

```diff
@@ -90,25 +125,60 @@ export default class GeFormFieldPicklist extends LightningElement {
         this.recordTypeId = this.defaultRecordTypeId;
     }
 
-    @api
-    get recordTypeId() {
-        return this._recordTypeId;
+    // ================================================================================
+    // WIRE METHODS
+    // ================================================================================
+    @wire(getPicklistValues, {
+        fieldApiName: '$fullFieldApiName',
+        recordTypeId: '$recordTypeId'
+    })
+    wiredPicklistValues({ error, data }) {
+        if (data) {
+            this.picklistValues = [CONSTANTS.PICKLIST_OPTION_NONE, ...data.values];
+        }
+        if (error) {
+            console.error(error);
+        }
     }
 
-    set recordTypeId(id) {
-        this._recordTypeId = id || this.defaultRecordTypeId;
+    // ================================================================================
+    // PRIVATE METHODS
+    // ================================================================================
+    handleValueChange(event) {
+        event.stopPropagation();
+        this.value = event.detail.value;
```

Comment:
What do you think about adding a `displayValue` prop here to the detail object and including the RecordTypeName? 
(I think the built-in change event only sends the value, right?) Then the new `updateFormState` method in the renderer could just append to state and it would include both the RT val and displayVal without needed to look up the RTName in the renderer?

Reply:
As discussed, we'll hold off on making additional changes to the existing picklist flow.